webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelect.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { TypeAheadSelect as PfTypeAheadSelect } from 'patternfly-react';
+
+const TypeAheadSelect = props => <PfTypeAheadSelect {...props} />;
+
+export default TypeAheadSelect;
```

Comment:
I'm thinking this file is not necessary at all.  You can just import the component from `patternfly-react` in `index.js` and wherever else you need it.

Reply:
I tried this before but it broke - I think it has to do with there no longer being the TypeAheadSelect default export?

Reply:
Just tried it and seems to work for me.  In `patternfly-react` it's a named export and not a default export, so I had to change 
```js
// index.js
import TypeAheadSelect from './TypeAheadSelect';
```
to
```js
// index.js
import { TypeAheadSelect } from 'patternfly-react';
```

Reply:
I tried exactly that a few times. The main difference was I didn't delete TypeAheadSelect.js but rather commented out the contents. Not sure if that would have affected it but it's all I can think of!